addons/account/views/report_payment_receipt.xml

```diff
@@ -0,0 +1,60 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Rename to "main_model"_"report_name"_templates.xml to put in report folder. See guidelines for more data.